library/core/src/num/f128.rs

```diff
@@ -1268,8 +1268,9 @@ impl f128 {
             min <= max,
             "min > max, or either was NaN",
             "min > max, or either was NaN. min = {min:?}, max = {max:?}",
-            min: f128,
-            max: f128,
+            // FIXME(f16_f128): Passed by-ref to avoid codegen crashes
```

Comment:
Is this needed since it looks like it should still get `#[inline]`?

Reply:
The "runtime panic" function generated here will not get `inline`. That's why this is needed.

Reply:
What am I missing - the non-`#[noinline]` version seems to still get it.

```rust
            else
                #[inline] // avoid the overhead of an extra fn call
                $(#[$runtime_attr])*
                $runtime
```

If this isn't accurate then I think we need to change things to avoid breaking Cranelift (@bjorn3)

Reply:
`const_assert!` uses `const_panic!` which uses `#[noinline]`. That's the entire point of this PR -- inlining the panic path is what caused the performance regression that this is fixing.